l10n_br_fiscal/models/document_line.py

```diff
@@ -121,43 +69,31 @@ def _operation_domain(self):
     # Amount Fields
     amount_untaxed = fields.Monetary(
```

Comment:
Is it still necessary to declare this fields again here? The only difference between them and the original in the mixin is the default value isn't it? If it's still necessary to put the default values here and not in the mixin, could it be just the default value, to make the code easier to maintain?
Example:
```
amount_untaxed = fields.Monetary(
    default=0.00
)
```